AY2223S1-CS2103T-T12-2/tp
# Feat: Add select contact command and UI

## Issues
- Resolves #67 

## Testing
- run app
- type `list_contact`
- type `add_task n/test c/1`
- type `select_contact 1`
- you should see your associated contact and tasks be selected
-`edit/delete` the task you should see the task hotupdate and be deleted / edited immediately

## Screenshots
<img width="1195" alt="Screenshot 2022-10-23 at 5 01 20 AM" src="https://user-images.githubusercontent.com/29945147/197362313-d38d51b4-0eaf-4cf1-81c3-41adf51c3a3c.png">
